recipes/earth2mip/meta.yaml

```diff
@@ -0,0 +1,90 @@
+{% set name = "earth2mip" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NVIDIA/earth2mip/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a49d0607893013783d30bfcb2f80412014ab535fbcc1e96dd139b78819bd98ab
+
+build:
+  noarch: python
+  script:
+    - sed -i.bak '/"pytest/d' pyproject.toml
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  entry_points:
+    - fcn = earth2mip.networks.fcn:load
+    - dlwp = earth2mip.networks.dlwp:load
+    - pangu = earth2mip.networks.pangu:load
+    - pangu_6 = earth2mip.networks.pangu:load_6
+    - pangu_24 = earth2mip.networks.pangu:load_24
+    - fcnv2_sm = earth2mip.networks.fcnv2_sm:load
+    - graphcast = earth2mip.networks.graphcast:load_time_loop
+    - graphcast_operational = earth2mip.networks.graphcast:load_time_loop_operational
+    - graphcast_small = earth2mip.networks.graphcast:load_time_loop_small
```

Comment:
Note that these are not script entry points, but rather plugin-type (see [here](https://github.com/NVIDIA/earth2mip/blob/18482abdd7be3acaa0c9b5b400260bae582ed3a8/pyproject.toml#L90)). Do we still need to list them here?

Reply:
Huh. I didn't notice it. This was created by grayskull and I didn't look thoroughly. Will fix.